src-tool/Pact/Analyze/Feature.hs

```diff
@@ -1651,6 +1652,20 @@ doc FTypeof = Doc
         (TyCon str)
   ]
 
+doc FContinue = Doc
```

Comment:
My addition, remove, I don't think it's necessary

Reply:
REmoved